Source/Icebreaker/Helpers/AdaptiveCards/PairUpNotificationAdaptiveCard.cs

```diff
@@ -0,0 +1,95 @@
+﻿//----------------------------------------------------------------------------------------------
```

Comment:
Where is this used now?

Reply:
This came in when I merged master into this branch @aosolis, so I have to delete this because this used the JSON payload and the C# class